features/test_runner/fastlane_test_runner_helpers/fastlane_ci_output.rb

```diff
@@ -0,0 +1,197 @@
+module FastlaneCI
+  # Implementation for the FastlaneCore::Interface abstract class
+  # this is set to the fastlane runner before executing the user's Fastfile
+  class FastlaneCIOutput < FastlaneCore::Interface
+    # the file path to the log file for the output
+    attr_accessor :file_path
+
+    # The block that's being called for each new line
+    # that should be printed out to the user
+    attr_accessor :block
```

Comment:
would it make sense to make this variable name more descriptive?

Reply:
Yep, will update

Reply:
`each_line_block`? or do you have something better in mind?

Reply:
that sounds good to me 👍